engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/engine/embedded/requestresponse/RequestResponseDeploymentStrategy.scala

```diff
@@ -31,12 +31,12 @@ object RequestResponseDeploymentStrategy {
   import net.ceedubs.ficus.readers.ArbitraryTypeReader._
 
   def apply(config: Config)(implicit as: ActorSystem, ec: ExecutionContext): RequestResponseDeploymentStrategy = {
-    new RequestResponseDeploymentStrategy(config.rootAs[RequestResponseConfig])
+    new RequestResponseDeploymentStrategy(config.rootAs[HttpConfig], config.rootAs[RequestResponseConfig])
```

Comment:
Maybe we should make the same layout of config as we have in docker? Am I right that it is now:
```
http { interface,port }
request-response { definitionMetadata }
```
?

Reply:
Unfortunately it requires entry in migration guide